core/src/main/scala/org/bitcoins/core/api/wallet/WalletApi.scala

```diff
@@ -278,6 +278,20 @@ trait WalletApi extends StartStopAsync[WalletApi] {
     } yield tx
   }
 
+  def sweepWallet(address: BitcoinAddress)(implicit
+      ec: ExecutionContext): Future[Transaction] = sweepWallet(address, None)
+
+  def sweepWallet(address: BitcoinAddress, feeRateOpt: Option[FeeUnit])(implicit
+      ec: ExecutionContext): Future[Transaction] = {
+    for {
+      feeRate <- determineFeeRate(feeRateOpt)
+      tx <- sweepWallet(address, feeRate)
+    } yield tx
+  }
+
+  def sweepWallet(address: BitcoinAddress, feeRate: FeeUnit)(implicit
```

Comment:
So.... what is the difference between `emptyWallet()` and `sweepWallet()` ? Neither of them have scaladocs on them

Reply:
Oh was the same thing, removed `emptyWallet` as we already have `isEmpty` so could be confusing